src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java

```diff
@@ -12,7 +12,7 @@
     @MessageMapping("/main.start")
     @SendTo("/game")
     public GameState start() {
-        return new GameState();
+        return new GameState(1);
```

Comment:
Magic number should be extracted to a constant

Reply:
are u on main branch?

Reply:
Ty chto tut pisheh?

Reply:
kto?

Reply:
Roma